src/features/account/components/AccountInfo.test.tsx

```diff
@@ -5,19 +5,23 @@ import { AccountInfo } from './AccountInfo';
 import { createStore } from '../../../store';
 import { render, VALID_ACCOUNT_1 } from '../../../utils/test-utils';
 
-jest.mock('../../../hooks/useTypedSignTrustCredential', () => ({
-  useTypedSignTrustCredential: () => ({
-    submitTypedSignRequest: jest.fn(),
-    isLoading: false,
-    isVerified: false,
-    payload: {},
-    signatureError: null,
+jest.mock('../../../hooks/useVerifiableCredential', () => ({
+  ...jest.requireActual('../../../hooks/useVerifiableCredential'),
+  useVerifiableCredential: () => ({
+    issuerAddress: 'issuerAddress',
+    signMessage: jest.fn(),
+    signError: null,
   }),
 }));
 
 jest.mock('wagmi', () => ({
   useEnsName: jest.fn(),
   useAccount: jest.fn(),
+  usePublicClient: jest.fn(),
+  useChainId: jest.fn(),
+  useSignTypedData: () => ({
+    signTypedDataAsync: jest.fn(),
+  }),
```

Comment:
may i ask why you need this? the useVerifiableCredential has mocked

Reply:
The code here was done before the rebase, previously the `useVerifiableCredential` hasn't been mocked and throw errors in wagmi, i have removed the code here after rebase.